Zend/zend_language_scanner.l

```diff
@@ -1250,6 +1258,69 @@ static void copy_heredoc_label_stack(void *void_heredoc_label)
 	zend_ptr_stack_push(&SCNG(heredoc_label_stack), (void *) new_heredoc_label);
 }
 
+/* Check that { }, [ ], ( ) are nested correctly */
+static void report_bad_nesting(char opening, int opening_lineno, char closing)
```

Comment:
I didn't prefix these functions with `zend_` because they are `static`. Is this OK? Not OK?

Reply:
That's fine, yes.